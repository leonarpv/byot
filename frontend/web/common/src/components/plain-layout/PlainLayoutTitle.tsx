import * as React from 'react';
import {Box, Typography, WithStyles} from '@material-ui/core';

interface Props {
  children: React.ReactNode;
}

export const PlainLayoutTitle: React.FC<Props> = (props: Props) => {
  return (
    <Box mb={10}>
      <Typography variant="h3">{props.children}</Typography>
    </Box>
  );
};
