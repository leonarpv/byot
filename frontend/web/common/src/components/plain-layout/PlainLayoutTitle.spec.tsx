import * as React from 'react';
import {render} from '@testing-library/react';
import {PlainLayoutTitle} from './PlainLayoutTitle';

describe('<PlainLayoutTitle/>', () => {
  it('should render', () => {
    const {container} = render(<PlainLayoutTitle>Title</PlainLayoutTitle>);
    expect(container).toMatchSnapshot();
  });
});
