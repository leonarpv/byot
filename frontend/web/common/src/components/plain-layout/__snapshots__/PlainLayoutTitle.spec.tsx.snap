// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlainLayoutTitle/> should render 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-1"
  >
    <h3
      class="MuiTypography-root MuiTypography-h3"
    >
      Title
    </h3>
  </div>
</div>
`;
